app/views/user_mailer/delete_work_notification.text.erb

```diff
@@ -1,9 +1,12 @@
 <% content_for :message do %>
-Dear <%= @user.default_pseud.byline %>,
-  
-Your work "<%= @work.title.html_safe %>" was deleted at <%= (@work.pseuds.count > 1 && @user != User.current_user) ? "the request of " + text_pseud(User.current_user.default_pseud) : "your request" %>.
-  
-If you have questions, please contact Support at <%= root_url %>support.
-  
-Attached is a copy of your work for your reference.
-<% end %>
\ No newline at end of file
+<%= t('.hello', user:  @user.default_pseud.byline) %>
+<%= (@work.pseuds.count > 1 && @user != User.current_user) ?
+t('.part1_other',
+        title: @work.title,
+        pseud: User.current_user.default_pseud ) :
+t('.part1_yourself',
+        title: @work.title) %>
+<%= t('.part2', support: t('user_mailer.delete_work_notification.support')+' '+root_url+"support" ) %>
```

Comment:
I think this could be `new_feedback_report_url` or `support_url` instead. Also, does this need to be the full key, or can it be shortened to just `t('.support')`?